ignite/cmd/generate_vuex.go

```diff
@@ -38,7 +38,7 @@ func generateVuexHandler(cmd *cobra.Command, args []string) error {
 	}
 
 	s.Stop()
-	fmt.Println("⛏️  Generated vuex stores.")
+	fmt.Println("⛏️  Generated Typescript Client and Vuex stores")
```

Comment:
We should also use the `cliui` session for the spinner and printing the feedback, similar to `ignite/cmd/generate_typescript_client.go`